README.md

```diff
@@ -25,6 +25,8 @@ the "boot JDK" which should generally be one major version prior to the one
 you are building (although one of the same major version will also work).
 Note that the build variant defaults to HotSpot if omitted which builds from the same repositories as Temurin.
 
+If you're using wsl, use the `--wsl` flag (ie: `./makejdk-any-platform.sh -J /usr/lib/jvm/java-21-openjdk-amd64 --wsl --create-sbom jdk21`).
```

Comment:
I would prefer to keep #4121 in it's own PR, so changes are indepenedent

Reply:
the idea was that hopefully #4121 would go first, and then the diff of this PR would just be the changes here (will update branch). When they go in, they would be independent changes (I'm assuming squash-and-merge policy), so can be independently reverted or cherrypicked

Reply:
the diff is now updated, so if it's squashed and merged only the relevant changes will be in this pr